.gitignore

```diff
@@ -42,3 +42,8 @@ next-env.d.ts
 
 # test subset config
 packages/test-harness/testSubsetGrep.properties
+
+data/playground/.metals
+data/playground/.vscode
```

Comment:
what is this one for?

Reply:
My mistake, I meant to ignore them in vscode